### PR TITLE
Auto-Merge setup via Mergify

### PR DESCRIPTION
**Pull request by W3stside**
Using mergify setup from dex-services

Merges when actions are green, base branch is ~`master`~ `develop`

~@anxolin base branch bring `master` is ok?~

### CODE REVIEW

**Review by alfetopito — COMMENTED** [review]
I think we should be based on `dev` since that's where we work and will be easier to see if something breaks that's not caught by the tests.

File: .mergify.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+pull_request_rules:
+  - name: Merge approved and green PRs with merge-when-green label
+    conditions:
+      - "#approved-reviews-by>=1"
+      - status-success=Travis CI - Pull Request
```
Comment (anxolin) [inline .mergify.yaml:5]: Why travis? we us GH now

File: .mergify.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      - label=merge when green
+    actions:
+      merge:
+        method: squash
+        strict: smart+fasttrack
+        commit_message: title+body
+  - name: automatic merge for Dependabot pull requests
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
+      - status-success=Travis CI - Pull Request
```
Comment (anxolin) [inline .mergify.yaml:16]: no travis

File: .mergify.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      - base=develop
+    actions:
+      merge:
+        method: squash
+        strict: smart+fasttrack
+        commit_message: title+body
```
Comment (anxolin) [inline .mergify.yaml:22]: Does it add a break line between the title and the body?

**Review by anxolin — COMMENTED** [review]
Awesome! thanks @W3stside

**Comment by anxolin** [comment]
Once it works, can u set it up in https://github.com/gnosis/gp-swap-ui too pls?

File: .mergify.yaml (hunk 1 above)
Comment (W3stside) [reply .mergify.yaml:5]: yup, will fix. Actually this whole section i am removing as it's ALL PRs, and I just want Dependabot

File: .mergify.yaml (hunk 2 above)
Comment (W3stside) [reply .mergify.yaml:16]: :+1:

File: .mergify.yaml (hunk 3 above)
Comment (W3stside) [reply .mergify.yaml:11]: need to check

**Review by alfetopito — APPROVED** [review]

File: .mergify.yaml (hunk 3 above)
Comment (W3stside) [reply .mergify.yaml]: i'll check against one of the open dependabot PRs

**Review by anxolin — APPROVED** [review]
Great!

**Pull request merged**